### PR TITLE
DOCS-3179 Browser RUM Data Collected Edits + Preview Link

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Addresses copy nits in the Data Collected doc page + pulls two other preview branches.

### Motivation

DOCS-3179

### Preview

https://docs-staging.datadoghq.com/alai97/rum-data-collected-documentation-update-preview/real_user_monitoring/browser/data_collected/

### Additional Notes

Reminder to myself to change the preview branches back to `develop` before merging!

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/real_user_monitoring/browser/data_collected.md [hunk 1]
```diff
@@ -26,16 +26,25 @@ further_reading:
...
 
-This page provides descriptions of each of the metrics and attributes collected. For guidance on what you can do with this data, see:
-- [Modifying Data and Context][1]
-- [Monitoring Page Performance][2]
-- [Monitoring Resource Performance][3]
-- [Collecting Browser Errors][4]
-- [Tracking User Actions][5]
+| Event Type     | Retention | Description                                                                                                                                                                                                                                                   |
+|----------------|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Session   | 30 days   | A user session begins when a user starts browsing the web application. It contains high-level information about the user (browser, device, geo-location). It aggregates all RUM events collected during the user journey with a unique `session.id` attribute. |
```
Comment (alai97) [inline content/en/real_user_monitoring/browser/data_collected.md:37]:
```suggestion
| Session   | 30 days   | A user session begins when a user starts browsing the web application. It contains high-level information about the user (browser, device, geo-location). It aggregates all RUM events collected during the user journey with a unique `session.id` attribute. **Note:** The session resets after 15 minutes of inactivity. |
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
No copy edits to make, but two things:
- should probably undo the changes to pull_config_preview before we merge, and
- There's a broken link being reported by the build

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -551,7 +551,7 @@
               text: "Explore your services, resources, and traces"
 
     - action: pull-and-push-folder
-      branch: master
+      branch: alai97/rum-data-collected-docs-update
```
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:554]:
```suggestion
      branch: master
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 3]
```diff
@@ -306,10 +306,10 @@
               tag: "Github"
               text: "dd-sdk-android Source code"
             - link: "/real_user_monitoring"
-              tag: "Homepage"
+              tag: "Documentation"
               text: "Explore Datadog RUM"
     - action: pull-and-push-file
-      branch: master
+      branch: alai97/rum-data-collected-docs-update
```
Comment (kayayarai) [inline local/bin/py/build/configurations/pull_config_preview.yaml:312]:
```suggestion
      branch: alai97/master
```

**Review by kayayarai — APPROVED** [review]
approved if you use these changes

**Pull request merged**